src/en/manhwaz/build.gradle

```diff
@@ -1,9 +1,9 @@
 ext {
     extName = 'ManhwaZ'
-    extClass = '.ManhwaZ'
-    themePkg = 'madara'
+    extClass = '.ManhwaZCom'
+    themePkg = 'manhwaz'
     baseUrl = 'https://manhwaz.com'
-    overrideVersionCode = 0
+    overrideVersionCode = 34
```

Comment:
```suggestion
    overrideVersionCode = 35
```

Reply:
ManhwaZ is currently 1.4.34, so baseVersionCode (1) + overrideVersionCode (34) will make it 35.